### PR TITLE
Update Chinese Language & Add some missing translations

### PR DESCRIPTION
**Pull request by sjqlwy**
feat: Update Chinese Language & Add some missing translations

Fluidd is a nice webUI for Klipper, however, when using in zh_CN language, some terms are confusing and missed. for example, `File` is used for filename and lefttime calculated by duration/percent, `All` is used for all_camera and home_all, they are just ok in English, but need to distinguish in Chinese, maybe in others language.
File and FileTime
Signed-off-by: sjqlwy, sjqlwy@gmail.com

Add some missing translations Like `Add Category`、`MOTORS OFF`、`QGL`, etc.
sorry I'm not that familiar with github, xD

### CODE REVIEW

**Review by pedrolamas — DISMISSED** [review]
LGTM

I've linted the code and made a few small changes to maintain consistency.

**Review by pedrolamas — DISMISSED** [review]

**Review by matmen — DISMISSED** [review]

File: src/locales/en.yaml [hunk 1]
```diff
@@ -168,6 +169,7 @@ app:
       upload_print: Upload & Print
       view: View
       reset_stats: Reset Stats
+      home_all: HomeAll
```
Comment (matmen) [inline src/locales/en.yaml:172]:
```suggestion
      home_all: All
```

though i don't really see a need to add another key just for this case..

File: src/components/widgets/status/StatusTab.vue [hunk 2]
```diff
@@ -105,7 +105,7 @@
 
               <status-label
                 v-else
-                :label="$t('app.general.label.file')"
+                :label="$t('app.general.label.percent_time')"
```
Comment (matmen) [inline src/components/widgets/status/StatusTab.vue:108]:
```suggestion
                :label="$t('app.general.label.file_time')"
```

File: src/locales/cn.yaml [hunk 3]
```diff
@@ -204,27 +209,48 @@ app:
...
+      apply_z_offset: 保存并应用 Z偏移设置
+      edit_filter: 编辑过滤器
+      heaters_busy: 正在打印中，关闭此加热器可能会导致打印失败
+      'on': 开启
+      unsaved_changes: 改动未保存
+      pressure_advance: 压力补偿
+      smooth_time: 平滑时间
+      edit_category: 编辑分类
+      add_category: 添加分类
+      percent_time: 预计剩余
```
Comment (matmen) [inline src/locales/cn.yaml:224]:
```suggestion
      file_time: 预计剩余
```

File: src/locales/en.yaml [hunk 4]
```diff
@@ -253,6 +257,9 @@ app:
       unsaved_changes: Unsaved Changes
       alias: Alias
       apply_z_offset: Apply and Save Z_Offset
+      edit_category: Edit category
+      add_category: Add category
+      percent_time: File
```
Comment (matmen) [inline src/locales/en.yaml:262]:
```suggestion
      file_time: File
```

File: src/components/widgets/toolhead/ToolheadMoves.vue [hunk 5]
```diff
@@ -39,7 +39,7 @@
           small-icon
           @click="sendGcode('G28', waits.onHomeAll)"
         >
-          {{ $t('app.general.btn.all') }}
+          {{ $t('app.general.btn.home_all') }}
```
Comment (matmen) [inline src/components/widgets/toolhead/ToolheadMoves.vue:42]: shouldn't this go under `app.tool.btn.home_all`?

**Review by matmen — CHANGES_REQUESTED** [review]

File: src/locales/en.yaml (hunk 1 above)
Comment (sjqlwy) [reply src/locales/en.yaml:172]: as mentioned above, `All` is used for all_camera and home_all, I wanna to keep the style consistent with `homeX` `homeY` `homeAll`(insteda of `All`) when translated into Chinese, they are just ok in English, but need to distinguish in Chinese, maybe in others language.

File: src/components/widgets/toolhead/ToolheadMoves.vue (hunk 5 above)
Comment (sjqlwy) [reply src/components/widgets/toolhead/ToolheadMoves.vue:42]: hah, yep!

**Comment by matmen** [comment]
Looks like this still remains unchanged, is there a reason for that? https://github.com/fluidd-core/fluidd/pull/607#discussion_r833641263

**Comment by pedrolamas** [comment]
@sjqlwy seems the PR is in a weird state with unresolved conflicts but it is not showing any changes at the moment... can I ask that you fix the conflicts so we can revise it again? Thank you!

**Comment by eduncan911** [comment]
> @sjqlwy seems the PR is in a weird state with unresolved conflicts but it is not showing any changes at the moment... can I ask that you fix the conflicts so we can revise it again? Thank you!

Unfortunately, this is one of the problems with the `pull` bot - it, by default, will do a Force Push of the remote repo into your user repo - **_which wipes out all of your commits in your branch_**.  

It's why I don't advise people use the `pull` bot to keep your forks updated: it actually erases anything you do.

I'm guessing, @sjqlwy is going to have to branch off of their local changes (so they keep their changes), delete/wipe out their local branch that matches the one in this PR, and then merge changes from their other branch back into this one.  That should get the commits back.

**Comment by pedrolamas** [comment]
This PR has NOT been merged, so I'm not sure what has happened here... I have no idea how to reopen this ticket...

**Pull request merged**